src/main/webapp/WEB-INF/tags/student/feedbackSubmissionEdit/response.tag

```diff
@@ -0,0 +1,42 @@
+<%@ tag description="questionWithResponses.tag - Display question with responses" %>
+<%@ tag import="teammates.common.util.Const"%>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+
+<%@ attribute name="questionWithResponses" type="teammates.ui.template.StudentFeedbackSubmissionEditQuestionsWithResponses" required="true" %>
+<%@ attribute name="response" type="teammates.ui.template.FeedbackSubmissionEditResponse" required="true" %>
+<%@ attribute name="isSessionOpenForSubmission" type="java.lang.Boolean" required="true" %>
+
+<c:set var="isNumResponsesMax" value="${questionWithResponses.numOfResponseBoxes eq questionWithResponses.maxResponsesPossible}"/>
+<c:set var="isRecipientNameHidden" value="${questionWithResponses.question.recipientNameHidden}"/>
+
+<c:choose>
+    <c:when test="${isRecipientNameHidden}"><c:set var="divClassType" value="class=\"col-sm-12\""/></c:when>
+    <c:when test="${isNumResponsesMax}"><c:set var="divClassType" value="class=\"col-sm-10\""/></c:when>
+    <c:otherwise><c:set var="divClassType" value="class=\"col-sm-8\""/></c:otherwise>
+</c:choose>
+
+<br>
+<div class="form-group margin-0">
+    <div ${isNumResponsesMax ? "class=\"col-sm-2 form-inline\"" : "class=\"col-sm-4 form-inline\""}
+         ${isRecipientNameHidden ?  "style=\"display:none\"" : "style=\"text-align:right\""}>
```

Comment:
`condition ? 'This Is Text' : 'This Is Also Text'` seems to be the standard for this, so something like `'class="col-sm-2 form-inline"'` may be better than escaping all the quotation marks.